packages/now-client/src/utils/index.ts

```diff
@@ -36,12 +35,8 @@ export type DeploymentEventType = (typeof EVENTS_ARRAY)[number];
 export const EVENTS = new Set(EVENTS_ARRAY);
 
 export function getApiDeploymentsUrl(
-  metadata?: Pick<DeploymentOptions, 'version' | 'builds' | 'functions'>
+  metadata?: Pick<DeploymentOptions, 'builds' | 'functions'>
 ) {
-  if (metadata && metadata.version !== 2) {
-    return '/v3/now/deployments';
-  }
-
   if (metadata && metadata.builds && !metadata.functions) {
     return '/v10/now/deployments';
```

Comment:
Should this be `v12`?

Reply:
Nope, 2 lines below there is the `v12` route option, but for `builds` we need to use `v10`